pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -52,30 +52,50 @@ class RandInt(Module):
 
   def create_returnn_layer_dict(self, low, high, size, dtype=None) -> Dict[str, Any]:
     dtype = dtype or "int64"
-    assert isinstance(low, int)
-    assert isinstance(high, int)
-    assert all(isinstance(sz, int) for sz in size)
-    return {"class": "rand_int", "shape": size, "maxval": high, "minval": low, "dtype": dtype}
+    if isinstance(low, Tensor):
+      low = low.type(dtype)
+      low = self._get_input_layer_name(low)
+    if isinstance(high, Tensor):
+      high = high.type(dtype)
+      high = self._get_input_layer_name(high)
+    naming = Naming.get_instance()
+    call = naming.module_call_stack[-1]
+    assert call.module.module is self
+    # add dependency to get complete in feed_dict in Module.make_output_tensor_from_returnn
+    call.inputs_tensor_deps.append(naming.inputs[0])
+    source = set()
+    for sz in size:
+      if isinstance(sz, Tensor):
+        if naming.tensors[sz].is_size_value.originating_tensor is not None:
```

Comment:
There should be some error checking such as checking that `naming.tensors[sz].is_size_value` is set.